testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -29,25 +29,37 @@
 When(/^I stop salt-minion on "(.*?)"$/) do |minion|
   node = get_target(minion)
   ## Uyuni uses Salt Bundle. Package name is "venv-salt-minion"
-  pkgname = $product == 'Uyuni' ? "venv-salt-minion" : "salt-minion"
-  node.run("rc#{pkgname} stop", check_errors: false) if minion == 'sle_minion'
-  node.run("systemctl stop #{pkgname}", check_errors: false) if %w[ceos_minion ubuntu_minion kvm_server xen_server].include?(minion)
+  pkgname = $product == 'Uyuni' ? 'venv-salt-minion' : 'salt-minion'
```

Comment:
This will collide with a refactor from @vzhestkov , now we use a function use_salt_bundle , instead of check for Uyuni product only.

Reply:
ok, i'll ping Victor.

Reply:
Yes, this is definetely colliding with our changes. We will adjust our PR to also cover this case once we get latest changes from `master` into our branch.

Thanks for the note! :+1: